mutmut/__main__.py

```diff
@@ -506,6 +523,9 @@ def run_mutation(config, filename, mutation_id):
     if cached_status != UNTESTED:
         return cached_status
 
+    if config.pre_mutation:
+        subprocess.check_output(config.pre_mutation, shell=True)
```

Comment:
What will happen if any of the pre/post mutation process fails?

Reply:
Then it terminates mutmut. I think that makes sense as a default, otherwise it's pretty hard to debug the post/pre hooks.

Reply:
I guess the exception log from failing processes must be shown to the end user.

Reply:
Yea, that's what you get by default. In fact you'll get all the stdout and stderr of the called process so your pre/post hooks should probably be silent in the normal case to not destroy the output of mutmut.

Reply:
Awesome! 👍